src/helpers.js

```diff
@@ -319,7 +319,8 @@ export function useOnScreen(ref, rootMargin = '0px') {
 export const getFilteredURL = (url, connected_data_parameters = []) => {
   if (!connected_data_parameters?.length) return url;
   let decodedURL = decodeURIComponent(url);
-  const queries = decodedURL.match(/(<<)(.*?)*>>/g); //safari: don't use lookbehind
+  //lookahead assertion to ensure that at least one character exists between '<<' and '>>';
+  const queries = decodedURL.match(/<<(?!\s*>>)[^>]*>>/g); //safari: don't use lookbehind
```

Comment:
@claudiaifrim @nileshgulia1 

Should this be a valid string?
```
<<<>>>
```
As it passes the regex :smile: 

https://regex101.com/

Reply:
Also this one:
```
<<<>>
```